### PR TITLE
task3-3_v1_math_to_js

### PR DESCRIPTION
**Pull request by EvgenijVY**

### CODE REVIEW

File: task3-3_v1_math_to_js/task3-3_v1_math_to_js.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+do {
+    let inputValue = prompt('Введите угол в радианах', '0');
+    angle = calculateAngle(inputValue);
+} while (angle === undefined)
+if (angle !== null) {
+    alert(angle);
+}
+
+function calculateAngle(countRadian) {
+    if (countRadian === null) {
```
Comment (ikrasnikov) [inline task3-3_v1_math_to_js/task3-3_v1_math_to_js.js:13]: I did not get, why do you need this check

File: task3-3_v1_math_to_js/task3-3_v1_math_to_js.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        return null;
+    }
+
+    if (Number(countRadian) != countRadian) {
+        alert('Вводить нужно число!');
+        return undefined;
```
Comment (ikrasnikov) [inline task3-3_v1_math_to_js/task3-3_v1_math_to_js.js:19]: you do not need to write undefined, you can just return

**Review by ikrasnikov — COMMENTED** [review]
the algorithm does not allow input fractional number

File: task3-3_v1_math_to_js/task3-3_v1_math_to_js.js (hunk 1 above)
Comment (EvgenijVY) [reply task3-3_v1_math_to_js/task3-3_v1_math_to_js.js:13]: > the algorithm does not allow input fractional number

you need to use the dot as delimiter

Comment (EvgenijVY) [reply task3-3_v1_math_to_js/task3-3_v1_math_to_js.js:13]: It's input cancelation check

File: task3-3_v1_math_to_js/task3-3_v1_math_to_js.js (hunk 2 above)
Comment (EvgenijVY) [reply task3-3_v1_math_to_js/task3-3_v1_math_to_js.js:19]: fixed

**Comment by EvgenijVY** [comment]
> the algorithm does not allow input fractional number

You need to use the dot as delimeter

**Pull request merged**